test/TestUtilities/Test.Utility/Test.Utility.csproj

```diff
@@ -19,6 +19,21 @@
     <ProjectReference Include="$(NuGetClientsSrcDirectory)NuGet.CommandLine\NuGet.CommandLine.csproj" />
   </ItemGroup>
 
+  <Choose>
```

Comment:
I don't get what this change achieves. 

Seems unnecessary as I don't see any functional differences.

Reply:
i dont know why it changed this. But will revert as part of squash n merge.